_drafts/2021-03-15-data_science_timeseries_plotting_notebook_template.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Jupyter Notebook Templates for Data Science: Plotting Time Series"
+description: >
+  Jumpstart your time series visualizations with this Jupyter plotting notebook!
+image: /files/jupyter-library/jupiter_in_the_rearview_mirror.jpg
+image_alt: >
+  The planet Jupiter as seen by the departing Juno spacecraft.
+categories: 
+  - data-visualization
+  - data-science
+  - jupyter
+  - my-projects
+---
+
+{% capture file_dir %}/files/jupyter-library/{% endcapture %}
+
+I often have data where each row describes an event that happened at some
+time. The data might describe [a word that my son spoke for the first
+time][sons_language], or [a collision that happened in California][collision],
+or [the finishing place of a rider in the Tour de France][2020_tour]. Some
+questions I always want to answer with the data are:
+
+[sons_language]: {% post_url 2020-02-10-my_sons_language_development_comparison %}#development
+[collision]: {% post_url 2019-02-20-switrs_bicycle_crashes_by_date %}#crashes-per-week
+[2020_tour]: {% post_url 2020-10-16-2020_tour_de_france_plot %}#the-race-for-yellow
+
+- What does the distribution of these events look like in time? 
+- Are there more of events of Type A or Type B?
```

Comment:
Are "Type A" and "Type B" data science terms? Otherwise I'm not clear on how this bullet fits